package.json

```diff
@@ -70,7 +70,8 @@
     "karma-sourcemap-loader": "^0.3.4",
     "karma-spec-reporter": "0.0.16",
     "karma-webpack": "~1.5.0",
-    "mocha": "^2.1.0"
+    "mocha": "^2.1.0",
+    "webpack-bundle-tracker": "0.0.8"
```

Comment:
If we have a separate `package.json` in admin, assuming it is even possible, wouldn't this be better in there?

Also end lines with commas so that adding a line doesn't require "removing" the existing line. Makes cleaner version diffs.